server_v2.py

```diff
@@ -267,15 +287,27 @@ def execute_stmt(conn, sqls, stmt):
             if not want_rows:
                 continue
 
-            rows.append([
-                value_from_sqlite(prepared.column(col_i))
-                for col_i in range(col_count)
-            ])
+            cells = []
+            for col_i in range(col_count):
+                try:
+                    val = prepared.column(col_i)
+                except Exception as e:
+                    msg = f"Could not get column {col_i}: {e}"
+                    raise ResponseError(msg) from e
+                cells.append(value_from_sqlite(val))
+
+            rows.append(cells)
 
         affected_row_count = conn.total_changes() - changes_before
         last_insert_rowid = conn.last_insert_rowid()
     except c3.SqliteError as e:
-        raise ResponseError(str(e)) from e
+        raise ResponseError(e) from e
+    except ResponseError:
+        raise
+    except Exception as e:
+        logger.error("Unexpected error executing: %r, exc: %r", stmt, e, exc_info=e)
+        msg = f"stmt: {stmt!r}, exc: {e}"
+        raise ResponseError(msg) from e
```

Comment:
If an unexpected error happens, we should let it propagate: in the end, it will be logged by the aiohttp server. `ResponseError` is a recoverable and expected error, we should not convert unexpected errors to it.